README.md

```diff
@@ -1 +1,22 @@
 # Flower docker image
+
+This repository contains the `Dockerfile` to build an image of [flower](https://github.com/mher/flower). 
+Flower is a real-time monitor and web admin for Celery distributed task queue.
+
+This image as a pinned version of celery to match the version used in [Open edX](https://open.edx.org/) and
+to monitor specifically edx workers.
+
+This image contains the following dependencies:
+
+```
+celery==3.1.18
+flower==0.9.3
+celery-redis-sentinel==0.3.0
+redis==2.10.6
+```
```

Comment:
I would remove this and point to the requirements file as we will forget to update the README when we upgrade dependencies.